web/static/js/content_types/image_tray.js

```diff
@@ -1,4 +1,7 @@
 import React from 'react'
+import ospryBuilder from '../utilities/ospry'
```

Comment:
@jamonholmgren Is the ospryBuilder js large? If so, you could be including quite a bit of js people might not use. Not a huge deal, I guess, since most people are going to want image uploads.